api/resources/views/filled-form/show.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.app')
+
+@section('content')
+
+    <div class="container mt-4">
+        <h1>Filled {{ $form->jobForm->name }} form</h1>
```

Comment:
maybe put this in a card instead of a weird card-like structure? 
```
<div class="card">
  <div class="card-header">
    <h1> Header </h1>
  </div>
  <div class="card-body">
    {{content}
  </div>
</div>
```

Reply:
If i liek the style i would take the card approach

Reply:
done